net.solarnetwork.node.setup.web/web/js/graph-data.js

```diff
@@ -0,0 +1,237 @@
+//Author Robert Rewcastle
+//script to draw line graphs of datums with a wattage variable
+//this script will put the graphs inside an html tag with a graphpoint class which 
+//one must put in the html document before loading the script
+//requires d3 https://d3js.org/d3.v3.js
+
+//json field we are interested in
+var key = "watts";
```

Comment:
The `key` variable name is too vague in this context. Something like `datumPropName` would better convey the intention.